src/inference/include/openvino/runtime/remote_context.hpp

```diff
@@ -68,6 +74,7 @@ class OPENVINO_RUNTIME_API RemoteContext {
     friend class ov::IExecutableNetworkWrapper;
     friend class ov::ICompiledModel;
     friend class ov::CompiledModel;
+    friend class ov::autobatch_plugin::Plugin;
```

Comment:
we should avoid it

Reply:
Agree with Ilya, Why do you need this change?

Reply:
```suggestion
```